CatchMe/CatchMe/Source/ViewControllers/AddActionVC.swift

```diff
@@ -225,4 +234,33 @@ class AddActionVC: UIViewController {
         vc.modalTransitionStyle = .crossDissolve
         self.present(vc, animated: true, completion: nil)
     }
+    
+    @objc func touchupPhotoButton(_ sender: UIButton) {
+        imagePicker.sourceType = .photoLibrary
+        imagePicker.allowsEditing = true
+        present(imagePicker, animated: true, completion: nil)
+    }
+    
+    @objc func touchupDeletePhotoButton(_ sender: UIButton) {
+        // 나중에 버튼 이미지 이름 부분에 디자이너가 넘겨준 에셋을 넘겨줄 예정입니다
+        photoButton.setImage(UIImage(named: ""), for: .normal)
+        if photoButton.currentImage == UIImage(named: "") {
+            deletePhotoButton.isHidden = true
+        } else {
+            deletePhotoButton.isHidden = false
+        }
```

Comment:
```suggestion
        if photoButton.isEnabled {
            deletePhotoButton.isHidden = true
        } else {
            deletePhotoButton.isHidden = false
        }
```
요런건 어떨까요?
저 조건문이 어떤 행동인지 직관적이지 않기도 하고, named에 한 글자만 틀려도 오류가 날 수 있는 상황이 만들어지기 쉽상이라서 저렇게 쓰기 보다는 Boolean형식을 가져와서 사용하는 게 더 좋을 거 같아요!

photoButton이 어떻게 기능하는지 저도 잘 모르겠지만, 
photo가 들어왔을 때는 저 부분을 눌러서 ImagePicker를 띄우는 기능을 못하게 해야 한다고 생각해서 isEnabled로 조건을 짰습니다.! 혹시 더 자세한 설명을 원한다면 언제든 물어보셔요. 글로 설명하기에 너무 어렵스...

Reply:
요부분은 질문을 갈겨야 할 거 같군여..!
photoButton을 통해 ImagePicker를 띄우고 이미지 선택해주면 그 이미지가
photoButton의 이미지로 들어오는 식으로 짜서.. currentImage로 설정해줬어여

이따가 아요 스크럼 때 화면공유해서 보여주는 것이 나을 거 같숨당........끄갹